01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,6 @@
+// const { readFileSync, writeFileSync } = require('fs')
+const  { readFileSync } = require('fs')
```

Comment:
I think you left out the writeFileSync part of this homework assignment. You have reading down, so you're already halfway there!